app/components/participation-list/modal.js

```diff
@@ -1,20 +1,42 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
-import { tracked } from '@glimmer/tracking';
+import { tracked } from 'tracked-built-ins';
```

Comment:
What's the difference between glimmer tracking and tracked-built-ins?

Reply:
https://github.com/pzuraq/tracked-built-ins
Tracked-built-ins can track things like native Maps and Arrays
The tracked decorator they provide works like the normal decorator for normal values, but can then also be used to track the builtin objects